public/index.html

```diff
@@ -10,6 +10,16 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!-- <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.12.1/css/all.css"
+      crossorigin="anonymous"
+    /> -->
+    <link
```

Comment:
implement fontawesome from npm, remove this cdn implementation @anjalipaswan18

Reply:
ok sie